examples/js/loaders/GLTF2Loader.js

```diff
@@ -1642,11 +1668,16 @@ THREE.GLTF2Loader = ( function () {
 
 					materialType = DeferredShaderMaterial;
 
-					var technique = json.techniques[ material.technique ];
+					var extension = extensions[ EXTENSIONS.KHR_TECHNIQUE_WEBGL ];
```

Comment:
I'm thinking to do that when I remove the existing `json.target` code (see the Update in the PR comment)
So, adding some comments so far?